src/index.html

```diff
@@ -13,6 +13,10 @@
     />
```

Comment:
The `href` attribute in the link tag should link to a CSS file, not a SCSS file. SCSS is a preprocessor which needs to be compiled into CSS for the browser to understand it.